### ISSUE TITLE
Can't  play my mp4 files that I've recorded

### ISSUE THREAD

**Issue by SGS84** [nicolasgarnier/drive-music-player#15]
I can't play my mp4 files I've recorded.  I am accessing the mp4 file via Google drive, then right clicking on the mp4 file I want to play.  Then selecting Music Player for Google Drive.  It loads and then tries to play the file but can't.  I get a "sorry, an error occurred while attempting to play this song".

Can someone help?  This would be an incredibly useful app for me if I can get it to work.

Best,
Scott
scottstephens84@gmail.com

**Comment by nicolasgarnier** [comment]
Hey @SGS84 I;ve fixed this issue yesterday. Indeed there was an issue with .mp4 files. Sorry about that.

**Comment by shenkin** [comment]
Hi, I just installed Music Player for Google Drive, and find that it cannot play an MP4 created by BoldBeast music recorder for Android. QuickTime on my Mac can play the file. I presume that the version of Music Player that I just installed includes the fix nicolasgarnier made in June.... (???)

An attempt to play the file evokes a message that says "Sorry! We are unable to play this song. More...", but there seems to be no more.

I'm using the Chrome browser on a Mac running OS X 10.10.5.
